### PR TITLE
donkrage.regpack version 1.51

### PR DESCRIPTION
**Pull request by donkrage**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/97700)

### CODE REVIEW

File: manifests/d/donkrage/regpack/1.51/donkrage.regpack.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: donkrage.regpack
+PackageVersion: 1.51
+PackageLocale: en-US
+Publisher: donkrage
+PublisherSupportUrl: https://discord.gg/mB6DprqmR9
+PackageName: regpack
```
Comment (stephengillie) [inline manifests/d/donkrage/regpack/1.51/donkrage.regpack.locale.en-US.yaml:9]:
```suggestion
PackageName: regpack
PackageUrl: https://github.com/donkrage/regpack
```

**Comment by Trenly** [comment]
I get a 404 when testing

**Review by Trenly — CHANGES_REQUESTED** [review]
This application doesn't seem to write any entries to Control panel, so there is no way for winget to know if it is installed or not

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**